test/JDBC/expected/BABEL-3512.out

```diff
@@ -472,23 +484,20 @@ Query Text: EXEC babel_3512_proc_10
 SET babelfish_showplan_all OFF
 GO
 
--- Test conflicting hints
-CREATE PROCEDURE babel_3512_proc_conflict AS
+-- Test conflicting hints raises error, but doesn't block creation of the proc. 
```

Comment:
Seems like this is another bug in Babelfish. Please leave a comment explaining that.

Reply:
turns out this was due to an earlier issue in the implementation of this PR.  Now we currently reject creating this proc